ibis/tests/util.py

```diff
@@ -33,12 +31,15 @@ def assert_equal(left, right):
                                     .format(repr(left), repr(right)))
 
 
-def skip_if_invalid_operation(f):
+def skipif_unsupported(f):
     @wrapped(f)
     @functools.wraps(f)
     def wrapper(backend, *args, **kwargs):
         try:
             return f(backend, *args, **kwargs)
-        except (com.OperationNotDefinedError, com.UnsupportedBackendType) as e:
-            pytest.skip('{} using {}'.format(e, backend.__name__))
+        except (com.OperationNotDefinedError,
+                com.UnsupportedBackendType,
+                com.TranslationError,
+                NotImplementedError) as e:
```

Comment:
Are we sure that none of these skips are spurious?

Reply:
We can be sure after eliminating all of the TranslationError's and NotImplementedError.
For example multipledispatch raises NotImplementedError on missing signature (in our case missing implementation).

Reply:
cool